Doc/library/sqlite3.rst

```diff
@@ -199,31 +199,39 @@ Module functions and constants
 
 .. data:: PARSE_DECLTYPES
 
-   This constant is meant to be used with the *detect_types* parameter of the
-   :func:`connect` function.
+   Use this flag together with the *detect_types* parameter of :meth:`connect` to enable
+   parsing of declared types for each column returned.
+   The types are declared when the database table is created.
+   ``sqlite3`` will look up a converter function using the first word of the
+   declared type as the converter dictionary key.
+   For example, the following SQL code results in the following lookups:
 
-   Setting it makes the :mod:`sqlite3` module parse the declared type for each
-   column it returns.  It will parse out the first word of the declared type,
-   i. e.  for "integer primary key", it will parse out "integer", or for
-   "number(10)" it will parse out "number". Then for that column, it will look
-   into the converters dictionary and use the converter function registered for
-   that type there.
+   .. code-block:: sql
+
+      CREATE TABLE test(
+         i integer primary key,  ! will look up a converter named "integer"
+         p point,                ! will look up a converter named "point"
+         n number(10)            ! will look up a converter named "number"
+       )
+
+   This flag may be paired with :const:`PARSE_COLNAMES` using the ``|``
+   operator.
 
 
 .. data:: PARSE_COLNAMES
 
-   This constant is meant to be used with the *detect_types* parameter of the
-   :func:`connect` function.
+   Use this flag together with the *detect_types* parameter of :meth:`connect` to enable
+   parsing of column names in queries.
+   ``sqlite3`` will look for strings containing square brackets (``[]``),
```

Comment:
Hm, how about this?

```suggestion
   ``sqlite3`` will parse strings wrapped in square brackets (``[]``),
```

Reply:
Or maybe this:

```suggestion
   Column names must be wrapped in square brackets (``[]``),
```

Reply:
Agreed :+1: , that's much better—"containing" is definitely not the right word here, as opposed to "contained in", "wrapped in", etc., which I should have spotted on the first pass. I also updated my original just in case